Projects/RevenueCat/Project.swift

```diff
@@ -30,11 +30,7 @@ let allDeploymentTargets: DeploymentTargets = .multiplatform(
 let project = Project(
     name: "RevenueCat",
     organizationName: .revenueCatOrgName,
-    settings: .settings(
-        base: [:].automaticCodeSigning(devTeam: .revenueCatTeamID),
-        configurations: .xcconfigFileConfigurations,
-        defaultSettings: .recommended
-    ),
+    settings: .framework,
```

Comment:
I think the RevenueCat project does not need the code signing or the xcconfig files. Is that correct or am I missing something?

Reply:
Technically the frameworks do not, BUT this project has a host app inside that might need it if someone wants to run it manually.

I see two options:
1. Set specific settings for the project
2. Create a new project only for that

I think I prefer 1 for now, and if we experience any downsides, we can always split it

Reply:
I 100% agree. Thanks for the input! 🙌